unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/BaseLayer.cs

```diff
@@ -0,0 +1,221 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.Skybox
+{
+    [System.Serializable]
+    public class TextureLayer
+    {
+        public int slotID;                              // Slot ID
+        public LayerRenderType renderType;
+        public LayerType layerType;
+        public bool enabled;
+        public string nameInEditor;
+        public bool expandedInEditor;
+        public float timeSpan_start;
+        public float timeSpan_End;
+        public float fadingInTime = 1;
+        public float fadingOutTime = 1;
+
+        public float tintercentage;
+        public List<TransitioningFloat> renderDistance;
+
+        public Texture2D texture;
+        public Texture2D textureNormal;
+        public Cubemap cubemap;
+        public Gradient color;
+        public Vector2 tiling;
+
+        public Vector2 flipBookRowsAndColumns = Vector2.one;
+        public float flipbookAnimSpeed = 1;
+
+        // Offset
+        public List<TransitioningVector2> offset;
+
+        // satellite
+        public List<TransitioningVector2> satelliteWidthHeight;
+
+        // Rotations
+        public List<TransitioningFloat> rotation_float;
```

Comment:
```suggestion
        public List<TransitioningFloat> floatRotations;
```
Please rename to keep consistency with the rest of the code.